### PR TITLE
Normalize docs

### LINKED ISSUE #259: Make documentation consistent
Our auto-generated documentation and forms have inconsistencies with punctuation. It should be fixed along with any other oddities.

### PR DESCRIPTION
**Pull request by josephlewis42**
Normalized sentences in documentation. Fixes #259

### CODE REVIEW

File: brokerapi/brokers/cloudsql/common-definition.go [hunk 1]
```diff
@@ -84,25 +84,25 @@ var commonProvisionVariables = []broker.BrokerVariable{
 	{
 		FieldName: "disk_type",
 		Type:      broker.JsonTypeString,
-		Details:   "(only for 2nd generation instances) ",
+		Details:   "(only for 2nd generation instances)",
```
Comment (mattysweeps) [inline brokerapi/brokers/cloudsql/common-definition.go]: Should this have an enum field as well?

File: brokerapi/brokers/cloudsql/postgres-definition.go [hunk 2]
```diff
@@ -151,7 +151,7 @@ func init() {
 			{
 				FieldName: "max_disk_size",
 				Type:      broker.JsonTypeString,
-				Details:   "Maximum disk size in GB (10 minimum/default)",
+				Details:   "Maximum disk size in GB 10 is the minimum.",
```
Comment (mattysweeps) [inline brokerapi/brokers/cloudsql/postgres-definition.go]: Should this be 2 sentences?

File: brokerapi/brokers/storage/definition.go [hunk 3]
```diff
@@ -28,13 +28,13 @@ func init() {
 		Name: "google-storage",
 		DefaultServiceDefinition: `{
 	        "id": "b9e4332e-b42b-4680-bda5-ea1506797474",
-	        "description": "A Powerful, Simple and Cost Effective Object Storage Service",
+	        "description": "A Powerful, Simple and Cost Effective Object Storage Service.",
```
Comment (mattysweeps) [inline brokerapi/brokers/storage/definition.go]: Are these caps supposed to be here?

File: docs/customization.md [hunk 4]
```diff
@@ -563,7 +563,7 @@ For example:
 
 ### Google CloudSQL MySQL Custom Plans
 
-Generate custom plans for Google CloudSQL MySQL.
+Generate custom plans for Google CloudSQL MySQL..
```
Comment (mattysweeps) [inline docs/customization.md]: double dot

File: docs/customization.md [hunk 5]
```diff
@@ -726,7 +726,7 @@ For example:
 
 ### Google CloudSQL PostgreSQL Custom Plans
 
-Generate custom plans for Google CloudSQL PostgreSQL.
+Generate custom plans for Google CloudSQL PostgreSQL..
```
Comment (mattysweeps) [inline docs/customization.md]: double dot

File: docs/customization.md [hunk 6]
```diff
@@ -890,7 +890,7 @@ For example:
 
 ### Google Spanner Custom Plans
 
-Generate custom plans for Google Spanner.
+Generate custom plans for Google Spanner..
```
Comment (mattysweeps) [inline docs/customization.md]: double dot

File: docs/use.md [hunk 7]
```diff
@@ -910,3 +910,6 @@ Uses plan: `a42c1182-d1a0-4d40-82c1-28220518b360`
     "role": "storage.objectAdmin"
 }
 ```
+
```
Comment (mattysweeps) [inline docs/use.md]: Three newlines spaces? should be 1, correct?

Comment (mattysweeps) [reply docs/use.md]: I think the total is 4 now, actually.

File: brokerapi/brokers/api_service/definition.go [hunk 8]
```diff
@@ -34,13 +34,13 @@ func init() {
 		DefaultServiceDefinition: `
 		{
       "id": "5ad2dce0-51f7-4ede-8b46-293d6df1e8d4",
-      "description": "Machine Learning Apis including Vision, Translate, Speech, and Natural Language",
+      "description": "Machine Learning Apis including Vision, Translate, Speech, and Natural Language.",
```
Comment (craigatgoogle) [inline brokerapi/brokers/api_service/definition.go]: s/Apis/APIs
Here and elsewhere

File: brokerapi/brokers/bigtable/definition.go [hunk 9]
```diff
@@ -99,14 +99,14 @@ func init() {
...
-				Details:   "The name of the BigTable dataset",
+				Details:   "The name of the BigTable dataset.",
 			},
 		),
 		PlanVariables: []broker.BrokerVariable{
 			{
 				FieldName: "storage_type",
 				Type:      broker.JsonTypeString,
-				Details:   "Either HDD or SSD (see https://cloud.google.com/bigtable/pricing for more information)",
+				Details:   "Either HDD or SSD. See https://cloud.google.com/bigtable/pricing for more information.",
```
Comment (craigatgoogle) [inline brokerapi/brokers/bigtable/definition.go]: s/See/See:

File: brokerapi/brokers/cloudsql/postgres-definition.go (hunk 2 above)
Comment (craigatgoogle) [reply brokerapi/brokers/cloudsql/postgres-definition.go]: Or else a well placed comma could also work :)

**Comment by craigatgoogle** [comment]
Modulo matty's feedback as well :)

File: brokerapi/brokers/cloudsql/common-definition.go (hunk 1 above)
Comment (josephlewis42) [reply brokerapi/brokers/cloudsql/common-definition.go]: I need to investigate this one further, probably as part of #155 and #254. When I was looking at the docs, "ssd" wasn't even listed as a valid type, so I need to investigate what's going on there.

**Pull request merged**